### ISSUE TITLE
Feature Request: Chinese characters

### ISSUE THREAD

**Issue by ASC8384** [fifonik/FFMetrics#13]
FFMetrics is very nice，thank your work.

But when I calculate VMAF for a video file named in Chinese, something is wrong. VMAF score can't display correctly while PSNR and SSIM are fine.

In order to troubleshoot the video itself, I rename `称重达拉崩吧.mkv` to `1.mkv`. And everything is all right.

![图片](https://user-images.githubusercontent.com/29878345/106418391-280e6680-6491-11eb-9bed-5b3080ab6c5a.png)

I'm not sure if it is the filename(maybe encoding or charcater set, like UTF-8 and [GBK](https://en.wikipedia.org/wiki/GBK_(character_encoding))) problem.

FFMetrics.log shows 
> 2021-02-01 13:01:42	ffmpeg.exe -hide_banner -nostdin -i C:/Users/privacy/称重达拉崩吧.mkv -i C:/Users/privacy/称重达拉崩吧.mp4 -lavfi [0:v]settb=AVTB,setpts=PTS-STARTPTS[main];[1:v]settb=AVTB,setpts=PTS-STARTPTS[ref];[main][ref]libvmaf='eof_action=endall:n_threads=7:phone_model=0:pool=harmonic_mean:model_path=C\:/Green/Official/FFMetrics/vmaf-models/vmaf_v0.6.1.json:log_fmt=json:log_path=C\:/Users/privacy/Temp/FFMetrics/称重达拉崩吧.mkv_24955089-9c51-46e6-83da-1826f396bedd.log' -f null -

I just find `绉伴噸杈炬媺宕╁惂.mkv_24955089-9c51-46e6-83da-1826f396bedd.log` in `.../Temp/FFMetrics/`.

By the way, VMAF's gragh is not frame by frame like the other two. It only appears after all the calculations have been completed. Is this normal？

English is not my native language, please excuse my typing errors.

**Comment by fifonik** [comment]
Thanks for you feedback. I will investigate the path encoding issue.

As for the VMAF graph, this is expected as while calculating VMAF ffmpeg not returning current frame info and I can only read log at the end of its calculation.

**Comment by fifonik** [comment]
> I just find 绉伴噸杈炬媺宕╁惂.mkv_24955089-9c51-46e6-83da-1826f396bedd.log in .../Temp/FFMetrics/.

So the log was successfully created? Does it contain data? Could you share it?

**Comment by ASC8384** [comment]
> 
> 
> > I just find 绉伴吨杈炬媺宕╁惂.mkv_24955089-9c51-46e6-83da-1826f396bedd.log in .../Temp/FFMetrics/.
> 
> So the log was successfully created? Does it contain data? Could you share it?
> 
> P.S. Nevermind, I was able to re-produce the issue. Investigating.

The log was successfully created.

The content is almost following:
```json
{
  "version": "2.1.1",
  "frames": [
    {
      "frameNum": 0,
      "metrics": {
        "integer_motion2": 0.000000,
        "integer_motion": 0.000000,
        "integer_adm2": 1.000000,
        "integer_adm_scale0": 1.000000,
        "integer_adm_scale1": 1.000000,
        "integer_adm_scale2": 1.000000,
        "integer_adm_scale3": 1.000000,
        "integer_vif_scale0": 1.000000,
        "integer_vif_scale1": 1.000000,
        "integer_vif_scale2": 1.000000,
        "integer_vif_scale3": 1.000000,
        "vmaf": 97.428029
      }
    },
......
    {
      "frameNum": 5377,
      "metrics": {
        "integer_motion2": 0.000000,
        "integer_motion": 0.000000,
        "integer_adm2": 0.999813,
        "integer_adm_scale0": 0.992069,
        "integer_adm_scale1": 0.993565,
        "integer_adm_scale2": 1.000898,
        "integer_adm_scale3": 1.003119,
        "integer_vif_scale0": 0.729817,
        "integer_vif_scale1": 0.947953,
        "integer_vif_scale2": 0.979279,
        "integer_vif_scale3": 0.991551,
        "vmaf": 95.602335
      }
    }
  ],
  "pooled_metrics": {
    "integer_motion2": {
      "min": 0.000000,
      "max": 55.964397,
      "mean": 0.166773,
      "harmonic_mean": 0.017292
    },
    "integer_motion": {
      "min": 0.000000,
      "max": 101.327011,
      "mean": 0.768257,
      "harmonic_mean": 0.048413
    },
    "integer_adm2": {
      "min": 0.282792,
      "max": 1.091633,
      "mean": 0.981284,
      "harmonic_mean": 0.977262
    },
    "integer_adm_scale0": {
      "min": 0.500941,
      "max": 1.016712,
      "mean": 0.981477,
      "harmonic_mean": 0.980235
    },
    "integer_adm_scale1": {
      "min": 0.340015,
      "max": 1.055008,
      "mean": 0.976220,
      "harmonic_mean": 0.972567
    },
    "integer_adm_scale2": {
      "min": 0.200020,
      "max": 1.108617,
      "mean": 0.979567,
      "harmonic_mean": 0.974182
    },
    "integer_adm_scale3": {
      "min": 0.134902,
      "max": 1.118638,
      "mean": 0.982917,
      "harmonic_mean": 0.976971
    },
    "integer_vif_scale0": {
      "min": 0.020262,
      "max": 1.000000,
      "mean": 0.768573,
      "harmonic_mean": 0.752756
    },
    "integer_vif_scale1": {
      "min": 0.036440,
      "max": 1.005258,
      "mean": 0.934773,
      "harmonic_mean": 0.919630
    },
    "integer_vif_scale2": {
      "min": 0.040495,
      "max": 1.006747,
      "mean": 0.957852,
      "harmonic_mean": 0.944550
    },
    "integer_vif_scale3": {
      "min": 0.044499,
      "max": 1.009801,
      "mean": 0.965980,
      "harmonic_mean": 0.954188
    },
    "vmaf": {
      "min": 0.000000,
      "max": 100.000000,
      "mean": 91.607310,
      "harmonic_mean": 42.809502
    }
  },
  "aggregate_metrics": {
  }
}
```

**Comment by fifonik** [comment]
I was able to re-produce the issue and debug a little. 
Unfortunately? it looks like this is ffmpeg or libvmaf issue.
If I run from command line:
```
ffmpeg.exe -i C:/path/绉伴噸杈炬媺宕╁惂.m2ts -i C:/path/ref.m2ts -frames:v 250 -lavfi [0:v]null[main];[1:v]null[ref];[main][ref]libvmaf='model_path=J\:/path/to/vmaf_v0.6.1.json:log_fmt=json:log_path=C\:/Temp/绉伴噸杈炬媺宕╁惂.log' -f null -
```
Input file will be opened without issue and log is created, but when I check the dir the log file is named incorrectly. It is NOT named as "绉伴噸杈炬媺宕╁惂.log" in my system. As a result, ffmetrics is unable to open the file.

I can try to fix the issue by not using any part original file name at all when I generate log file name (just auto-generated UUID like "24955089-9c51-46e6-83da-1826f396bedd.log" should be fine as workaround).
Not really sure that I'm happy with this solution. 
I'll need to google a bit more and think about it.

**Comment by ASC8384** [comment]
What a pity!
I re-produced the issue with `ffmpeg.exe -i C:/C0007.MP4 -i C:/测试.mkv -lavfi [0:v]null[main];[1:v]null[ref];[main][ref]libvmaf='n_threads=4:model_path=C\:/vmaf_v0.6.1.json:log_fmt=json:log_path=C\:/测试.log' -f null -` in cmd.

And then I tested it in Python:

```python
In [1]:  str_src = '称重达拉崩吧'            
                                       
In [2]:  str_ref = '绉伴吨杈炬媺宕╁惂'         
                                       
In [3]:  stru = str_src.encode('utf8') 
                                       
In [4]:  print(stru.decode('gbk'))     
绉伴噸杈炬媺宕╁惂
```

**Comment by fifonik** [comment]
I hope it is fixed in 0.9.3b that I just published. 
Please check and let me know if you still have issues.

**Comment by ASC8384** [comment]
`0.9.3b` works great!
Thank you very much!
